hypervisor/arch/x86/cpu.c

```diff
@@ -446,7 +446,7 @@ void cpu_dead(void)
 		/* clean up native stuff */
 		vmx_off();
 		/* TODO: a cpu dead can't effect the RTVM which use Software SRAM */
-		cache_flush_invalidate_all();
+		wbinvd();
```

Comment:
why not "flush_invalidate_all_cache"?

Reply:
BTW, given that flush_tlb actually means flush and invalidate, I think we can use flush_cache_all here too.

Reply:
> why not "flush_invalidate_all_cache"?

IMHO, we should use wbinvd for x86 special code and use flush_invalidate_all_cache for non-arch related code

Reply:
> BTW, given that flush_tlb actually means flush and invalidate, I think we can use flush_cache_all here too.

Do you mean flush_cache_range(0, all address space) ?

Reply:
Yes

Reply:
x86 code call general code should be fine too.